### PR TITLE
Ws2.8 add sighting seed data

### PR DESCRIPTION
**Pull request by Purbai**
Created sample data for sightings:
1. Created seedSightings.cs
2. Amended program.cs 

Checked using PGAdmin that data was loaded in the Sightings table

### CODE REVIEW

File: backend/SeedData/SeedSightings.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                Id = 6,
+                UserId = 2,
+                WhaleSpeciesId = 1,
+                Latitude = 54.012354F,
+                Longitude = -94.012267F,
+                PhotoUrl =
+                    "https://images.unsplash.com/photo-1570913179118-f3d24be1d1f7?q=80&w=3943&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
+                Description = "Test Whale pic",
+                DateTime = DateTime.Parse("29Mar2023 13:42:00").ToUniversalTime(),
+                IsApproved = true
```
Comment (Kiran-R-K) [inline backend/SeedData/SeedSightings.cs:94]: good idea adding some approved and some not approved sightings

**Review by Kiran-R-K — CHANGES_REQUESTED** [review]
Good work - a couple of changes requested and I think there's been some merging issues so  let's remerge main into the branch together

File: backend/Program.cs [hunk 2]
```diff
@@ -69,7 +71,10 @@ public static async Task Main(string[] args)
                 var userManager = scope.ServiceProvider.GetRequiredService<UserManager<User>>();
                 await SampleUsers.CreateAdminAsync(userManager);
                 await SampleUsers.CreateUsersAsync(userManager);
-                
+
+                var seeder = scope.ServiceProvider.GetService<SeedSightings>();
```
Comment (Kiran-R-K) [inline backend/Program.cs:75]: can this be renamed to 'sightingsSeeder' inline with 'speciesSeeder' naming below

**Review by Kiran-R-K — CHANGES_REQUESTED** [review]
I've reviewed and added another comment - let me know if you have any questions

File: README.md [hunk 3]
```diff
@@ -28,6 +28,8 @@ npm install
 ```bash
 dotnet restore
 dotnet dotnet-ef database update
+dotnet run --seed (will add seed data)
+
```
[the remark this answers is not in the document]
Comment (Purbai) [reply README.md:32]: already resolved when did merge issue resolution

**Review by Kiran-R-K — APPROVED** [review]
well done - it's ready to go

**Pull request merged**